python_scripts/linear_models.py

```diff
@@ -640,7 +651,8 @@ def f(x):
 # %% [markdown]
 # Note that, we selected this alpha *without* using the testing set ; but
 # instead by using the validation set which is a subset of the training
-# data. This has been seen in the lesson *basic hyper-parameters tuning*.
+# data. So we did not "overfit" the test data.
+# This has been seen in the lesson *basic hyper-parameters tuning*.
```

Comment:
```suggestion
# data. This is so we do not "overfit" the test data and
# can be seen in the lesson *basic hyper-parameters tuning*.
```

Reply:
Maybe 'this is so that the we never use the test data when fitting our estimator" ?